pkgs/development/python-modules/functiontrace/default.nix

```
@@ -0,0 +1,27 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, toml
+}:
+
+buildPythonPackage rec {
+  pname = "functiontrace";
+  version = "0.3.7";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-3bnxZFq1/D9ntwfv7O2YU6MnKEDWWIG4zX0e3cgCleg=";
+  };
+
+  propagatedBuildInputs = [
+    toml
+  ];
```

```suggestion
  nativeBuildInputs = [
    setuptools
    toml
  ];
```

According to the `pyproject.toml`, `setuptools` and `toml` are only build-time deps and not runtime deps